src/modules/twap/utils/getTwapOrderStatus.ts

```diff
@@ -24,8 +24,11 @@ export function getTwapOrderStatus(
   return TwapOrderStatus.Scheduled
 }
 
-export function isTwapOrderExpired(order: TWAPOrderStruct): boolean {
-  const { t0: startTime, n: numOfParts, t: timeInterval } = order
+export function isTwapOrderExpired(order: TWAPOrderStruct, executionDate: Date | null): boolean {
+  if (!executionDate) return false
```

Comment:
The other way around?
If `executionDate` exists, the twap cannot be expired.
Assuming by `executionDate` the date of which the order has been executed.

If this means something else, please update the parameter name 🤞

Reply:
Sorry, I've confused you by naming. `executionDate` in this context means `startDate`.
Will rename it